lms.env.json

```diff
@@ -0,0 +1,179 @@
+                                                                                                                                                            {
```

Comment:
@clrux seems adding LMS configuration .json file here is not the right way of updating them. Please ask someone from devops, may be @e0d .